vm_scripts/take_snapshot.py

```
@@ -0,0 +1,347 @@
+                    if snapshot.childSnapshotList:
+                        found_snapshot = find_snapshot_in_tree(snapshot.childSnapshotList, target_name)
+                        if found_snapshot:
+                            return found_snapshot
+
+                return None
+
+            # Find the target snapshot
+            target_snapshot = find_snapshot_in_tree(snapshot_tree, snapshot_name)
+
+            if not target_snapshot:
+                logger_message = f"Snapshot not found to revert"
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)